task/sast-shell-check/0.1/README.md

```diff
@@ -0,0 +1,32 @@
+# sast-shell-check task
+
+## Description:
+
+The sast-shell-check task uses [shellcheck](https://www.shellcheck.net/) tool to perform Static Application Security Testing (SAST), a popular cloud-native application security platform.
+
+ShellCheck is a static analysis tool, gives warnings and suggestions for bash/sh shell scripts.
+
+## Params:
+
+| Name              | Description                                                                                  | Default Value | Required |
+|-------------------|----------------------------------------------------------------------------------------------|---------------|----------|
+| KFP_GIT_URL       | Git repository to download known false positives files from.                                 | ""            | No       |
+| PROJECT_NVR       | Name-Version-Release (NVR) of the scanned project, used to find path exclusions (optional).  | ""            | No       |
+| RECORD_EXCLUDED   | Whether to record the excluded findings (it is optional, true or false).<br/>If `true`, the the excluded findings will be stored in `record_excluded.json`.  | "false"            | No       |
```

Comment:
`it is optional, true or false` -> `defaults to false`

Please also change the file name to `excluded-findings.json` for consistency.

Reply:
Addressed